src/plugins/meter-circular/circular.js

```diff
@@ -0,0 +1,157 @@
+( function( $, window, wb ) {
+"use strict";
+
+var componentName = "wb-meter-circular",
+	selector = "meter." + componentName,
+	initEvent = "wb-init" + "." + componentName,
+	$document = wb.doc,
+	meterPrefix = "mtr-",
+	selectorValue = "." + meterPrefix + "val",
+	selectorLabel = "." + meterPrefix + "lbl",
+	clsOptimal = meterPrefix + "optimal",
```

Comment:
I'm guessing `cls` is a short form for `class`, but it doesn't make sense to short stuff that the minifier will correct